src/metpy/calc/thermo.py

```diff
@@ -45,7 +45,8 @@ def relative_humidity_from_dewpoint(temperature, dewpoint):
     -------
     `pint.Quantity`
         Relative humidity
-
+        
+    .. math:: rh = \frac{e(T_d)}{e_s}
```

Comment:
Can you also make explicit the dependency of `e_s` on temperature, maybe:
```suggestion
    .. math:: rh = \frac{e(T_d)}{e_s(T)}
```